ConsentViewController/Classes/ConsentViewController.swift

```diff
@@ -21,26 +21,26 @@ import Reachability
  ```
  var consentViewController: ConsentViewController!
  override func viewDidLoad() {
-     super.viewDidLoad()
-     consentViewController = ConsentViewController(accountId: <ACCOUNT_ID>, siteName: "SITE_NAME", stagingCampaign: true|false)
-     consentViewController.onMessageChoiceSelect = {
-        (cbw: ConsentViewController) in print("Choice selected by user", cbw.choiceType as Any)
-     }
-     consentViewController.onInteractionComplete = { (cbw: ConsentViewController) in
-         print(
-             cbw.euconsent as Any,
-             cbw.consentUUID as Any,
-             cbw.getIABVendorConsents(["VENDOR_ID"]),
-             cbw.getIABPurposeConsents(["PURPOSE_ID"]),
-             cbw.getCustomVendorConsents(),
-             cbw.getCustomPurposeConsents()
-         )
-     }
+ super.viewDidLoad()
+ consentViewController = ConsentViewController(accountId: <ACCOUNT_ID>, siteName: "SITE_NAME", stagingCampaign: true|false)
+ consentViewController.onMessageChoiceSelect = {
+ (cbw: ConsentViewController) in print("Choice selected by user", cbw.choiceType as Any)
+ }
+ consentViewController.onInteractionComplete = { (cbw: ConsentViewController) in
+ print(
+ cbw.euconsent as Any,
+ cbw.consentUUID as Any,
+ cbw.getIABVendorConsents(["VENDOR_ID"]),
+ cbw.getIABPurposeConsents(["PURPOSE_ID"]),
+ cbw.getCustomVendorConsents(),
+ cbw.getCustomPurposeConsents()
+ )
+ }
  view.addSubview(consentViewController.view)
  ```
-*/
```

Comment:
@vilasma can you do me a favour and clean the spacing differences? E.g. reseting the lines that don't have code changes

Reply:
Okay, I will clean the project also I will remove the methods which are commented by me.

Reply:
@vilasma it seems that the spaces issue remained. Could you have another look?